ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```diff
@@ -64,9 +78,17 @@ LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
 LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   packer@"${INSTANCE_PREFIX}" \
-  --command 'sudo yum install -y unzip'
+  --command "timeout 360m bash -ce \"/home/packer/run-tests.sh\" && echo \"### Fetching results\" && tar -czvf /tmp/artifacts.tar.gz /tmp/artifacts"
+
+LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
+  --quiet \
+  --project "${GOOGLE_PROJECT_ID}" \
+  --zone "${GOOGLE_COMPUTE_ZONE}" \
+  --recurse packer@"${INSTANCE_PREFIX}":/tmp/artifacts.tar.gz "${HOME}"/artifacts.tar.gz
+
+tar -xzvf "${HOME}"/artifacts.tar.gz -C "${ARTIFACT_DIR}"
 
 LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   packer@"${INSTANCE_PREFIX}" \
-  --command "timeout 360m bash -ce \"/home/packer/run-tests.sh\""
+  --command '/home/packer/rc.sh'
```

Comment:
Is this because we want to get back the exit code from `run-test.sh` which is going to be overwritten by the other comands? Can we have a comment somewhere making this explicit? Nothing seems to be creating `/tmp/test-return`, I suspect `run-test.sh` needs to be modified?